Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -138,6 +140,185 @@ public class PushDeploymentController : ApiController
             }
         }
 
+        [HttpPost]
+        [HttpPut]
+        public async Task<HttpResponseMessage> OnePushDeploy(
+            [FromUri] bool isAsync = false,
+            [FromUri] string author = null,
+            [FromUri] string authorEmail = null,
+            [FromUri] string deployer = OneDeploy,
+            [FromUri] string message = DefaultMessage)
+        {
+            using (_tracer.Step("OnePushDeploy"))
+            {
+                var response = Request.CreateResponse();
+
+                //Parses all query parameters into queryCollection
+                var queryCollection = Request.RequestUri.ParseQueryString();
+                var deployType = queryCollection[Constants.Type];
+                var website_framework = System.Environment.GetEnvironmentVariable(Constants.Framework).ToLower();
```

Comment:
>var website_framework = System.Environment.GetEnvironmentVariable(Constants.Framework).ToLower(); [](start = 16, length = 97)

Grouping all similar statements (queryCollection[...]) and then keeping the GetEnvironmentVariable line separate is good for readability